app/cni/cmd/install/validation.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/kumahq/kuma/pkg/util/files"
+)
+
+func lookForValidConfig(files []string, checkerFn func(string) error) (string, bool) {
+	for _, file := range files {
+		err := checkerFn(file)
+		if err != nil {
+			log.Error(err, "error occurred testing config file", "file", file)
```

Comment:
I don't think this should be an error,. `checkerFn` is a predicate. It's normal that some files may not fulfill this predicate